filters/auth/grantcookie_test.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 const (
-	TestToken        = "foobarbaz"
-	TestRefreshToken = "refreshfoobarbaz"
+	TestToken                = "foobarbaz"
+	TestRefreshToken         = "refreshfoobarbaz"
+	TestAccessTokenExpiresIn = time.Hour
```

Comment:
I wonder if you need all these exported or can you re-declare them in auth_test if you require them there, too?

Reply:
Re-declared them in auth_test and added a comment here that they should be kept in sync.